vendordeps/REV2mDistanceSensor.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
why would we use a distance sensor when we already have the color sensor that can determine distance